jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/common/BadgedButton.java

```diff
@@ -0,0 +1,12 @@
+package com.epam.jdi.light.ui.bootstrap.elements.common;
+
+import com.epam.jdi.light.elements.interfaces.base.HasBadge;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+
+/**
+ * To see an example of Badge web element please visit https://getbootstrap.com/docs/4.3/components/badge/#example
+ */
+
+public class BadgedButton extends Button implements HasBadge {
```

Comment:
Please, move this new class to "jdi-light-bootstrap-tests" package instead of regular jdi-light-bootstrap

Reply:
Would be moved later, in the process of refactoring a Button element